README.md

```diff
@@ -13,6 +13,15 @@ rspec-core and rspec-mocks):
 
     gem install rspec
 
+Want to run against the `master` branch? You'll need to include the dependent
+RSpec repos as well. Add the following to your `Gemfile`:
+
+```ruby
+%w[rspec-core rspec-expectations rspec-mocks rspec-support].each do |lib|
+  gem lib, :git => "git://github.com/rspec/#{lib}.git", :branch => 'master'
```

Comment:
You know, `:branch => 'master'` is unnecessary since `master` is the default branch.  Not a big deal that it's here, though.